pkg/qemu/qemu.go

```diff
@@ -920,11 +920,13 @@ func Cmdline(ctx context.Context, cfg Config) (exe string, args []string, err er
 	args = append(args, "-chardev", fmt.Sprintf("socket,id=%s,path=%s,server=on,wait=off", qmpChardev, qmpSock))
 	args = append(args, "-qmp", "chardev:"+qmpChardev)
 
+	// virtserialport doesn't seem to work reliably: https://github.com/lima-vm/lima/issues/2064
+	// should go together with hostagent::New changes
 	// Guest agent via serialport
-	guestSock := filepath.Join(cfg.InstanceDir, filenames.GuestAgentSock)
-	args = append(args, "-chardev", fmt.Sprintf("socket,path=%s,server=on,wait=off,id=qga0", guestSock))
-	args = append(args, "-device", "virtio-serial")
-	args = append(args, "-device", "virtserialport,chardev=qga0,name="+filenames.VirtioPort)
+	// guestSock := filepath.Join(cfg.InstanceDir, filenames.GuestAgentSock)
+	// args = append(args, "-chardev", fmt.Sprintf("socket,path=%s,server=on,wait=off,id=qga0", guestSock))
+	// args = append(args, "-device", "virtio-serial")
+	// args = append(args, "-device", "virtserialport,chardev=qga0,name="+filenames.VirtioPort)
```

Comment:
If the code is not needed lets delete it.

Reply:
I'm still using it in an on/off manner (with changes in hostagent.go), while working on QEMU support on Windows hosts. So, it is not 100% useless for sure. May be we can just dismiss this PR for now and I will later create some other more prepared with either removal or feature toggle.

Reply:
Maybe we need an option to enable/disable this?

You can also make this a draft, maybe others will find this change helpful.